Packs/Censys/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Censys
+- %%cen-search command enhancement that will fetch all the entries from the search not only first page of the results.%%
```

Comment:
```suggestion
- **cen-search** - adding pagination to handle search results with more than one page.
```
I removed the unnecessary ```%``` sign and added bolding to the command name (using ```**```) to help with the readability.